lib/tasks/convert_heic_documents_to_jpg.rake

```diff
@@ -0,0 +1,17 @@
+namespace :documents do
+  desc "Converts existing documents with heic attachments to jpg"
+  task convert_heic_to_jpg: [:environment] do
+    Document.find_each do |document|
+      if document.upload.filename.extension_without_delimiter.downcase == "heic"
+        puts "Converting document #{document.id}"
+        begin
+          document.convert_heic_upload_to_jpg!
+        rescue Expection => e
```

Comment:
I think `Expection` here is a typo for `Exception`. I'd like to see that fixed in the PR.

Non-blocking: Since it's a rake task and going to be executed manually, I prefer you remove the begin/rescue and allow the rake task to crash. I think we don't need to proceed through and process the other HEIC files if one raises an error; we can fix it or remove the image, then re-run the Rake task. I'm open to the existing behavior (work through all HEIC files that don't crash, printing out a message for those that crash) if you prefer.